src/node/install.sh

```diff
@@ -107,7 +107,7 @@ export NVM_SYMLINK_CURRENT=true
 if [ -d "${NVM_DIR}" ]; then
     echo "NVM already installed."
     if [ "${NODE_VERSION}" != "" ]; then
-       su ${USERNAME} -c ". $NVM_DIR/nvm.sh && nvm install ${NODE_VERSION} && nvm clear-cache"
+       su ${USERNAME} -c "umask 0002 && . $NVM_DIR/nvm.sh && nvm install ${NODE_VERSION} && nvm clear-cache"
```

Comment:
What is the `2` bit doing here in `umask 0002`

Reply:
Removed that, wasn't needed to fix the issue.